Source/WarpX.cpp

```diff
@@ -524,7 +524,7 @@ WarpX::ReadParameters ()
         int quantum_xi_is_specified = queryWithParser(pp, "quantum_xi", quantum_xi_tmp);
         if (quantum_xi_is_specified) {
             double const quantum_xi = quantum_xi_tmp;
-            quantum_xi_c2 = quantum_xi * PhysConst::c * PhysConst::c;
+            quantum_xi_c2 = static_cast<amrex::Real>(quantum_xi * PhysConst::c * PhysConst::c);
```

Comment:
Just double checking here: does the constant fit into a float for `quantum_xi_c2`?

Reply:
If I understand correctly, these lines allow a user to re-define the quantity `quantum_xi_c2` in order to enhance Quantum effects. The default value (the one which gives physically correct results) is:
```
    static constexpr auto xi_c2 = static_cast<amrex::Real>( 1.1728865132395492e-35 ); // This should be usable for single precision, though
    // very close to smallest number possible (1.2e-38)
```
So it should be fine to use a float here. Indeed, `quantum_xi_c2` is already a `Real`:
```
Real WarpX::quantum_xi_c2 = PhysConst::xi_c2;
```